src/arrayMethodJoin.js

```diff
@@ -5,7 +5,21 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let resultingString = '';
+    const splitter = separator === undefined ? ',' : separator;
+
+    for (let i = 0; i < this.length; i += 1) {
+      const item = this[i] === null || this[i] === undefined ? '' : this[i];
```

Comment:
o need to add empty string, check if your item is not null and undefined, and add it only in this case